master/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Security using Docker Labels and Calico Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+All traffic must be explicitly allowed through configuration of the globally
+defined policy which uses selectors to determine which subset of the policy is
+applied to each container based on their labels.  This approach provides a
+powerful way to group together all of your network Policy, makes it easy to
+reuse policy in different networks, and makes it easier to define policy that
+extends across different orchestration systems that use Calico.
+
+When Calico is configured to use container labels, profiles are not created and 
+have no impact on any container traffic.
+
+## Enabling Docker Networking Container Labels Policy
+
+To enable the container labels Policy with Docker Networking the flag
+`--use-docker-networking-container-labels` must be passed when starting
+calico/node with the `calicoctl node run` command.  All calico/node instances
+should be started with the flag to avoid a mix of labels and profiles.
+
+## Managing Calico policy for a network
+
+This section provides an example applying policy using the approach described
+above once container labels are enabled.
+
+We create a Calico-Docker network and use the `calicoctl` tool to set policies
+that achieve the required isolation and allowances.
+
+For the example let's assume that we want to provide the following isolation
+between a set of database containers and a set of frontend containers:
+
+-  Frontend containers can only access the Database containers over TCP to port 3306.
+   For now we'll assume no other connectivity is allowed to/from the frontend.
+-  Database containers have no isolation between themselves (to handle synchronization
+   within a cluster).  This could be improved by locking down the port ranges and
+   protocols, but for brevity we'll just allow full access between database
+   containers.
+
+### Global policy applied through label selection
+
+This example demonstrates using global selector-based policy with labels
+extracted from the Docker containers.
+
+#### 1. Create the Docker network
+
+On any host in your Calico / Docker network, run the following command:
+
+```
+docker network create --driver calico --ipam-driver calico-ipam net1
+```
+
+#### 2. Create the Labeled Workloads
+
+Create the workloads as docker containers with appropriate labels.
```

Comment:
With this line up here the user might think they're supposed to do something, but the command this is referencing is 3 paragraphs down. This line should probably just be moved right before the bash output below.